doc/services/device_mgmt/smp_groups/smp_group_0.rst

```diff
@@ -669,3 +671,112 @@ where:
     | "rc"         | :c:enum:`mcumgr_err_t`                        |
     |              | only appears if non-zero (error condition).   |
     +--------------+-----------------------------------------------+
+
+Bootloader Information
+**********************
+
+Allows to obtain information on board bootloader and its parameters.
+
+Bootloader Information Request
+==============================
+
+Bootloader information request header:
+
+.. table::
+    :align: center
+
+    +--------+--------------+----------------+
+    | ``OP`` | ``Group ID`` | ``Command ID`` |
+    +========+==============+================+
+    | ``0``  | ``0``        |  ``8``         |
+    +--------+--------------+----------------+
+
+CBOR data of request:
+
+.. code-block:: none
+
+    {
+        (str,opt)"query"  : (str)
+    }
+
+where:
+
+.. table::
+    :align: center
+
+    +--------------+-----------------------------------------------+
+    | "query"      | Is string representing query for parameters,  |
+    |              | with no restrictions how the query looks like |
+    |              | as processing of query is left for bootloader |
+    |              | backend.                                      |
+    +--------------+-----------------------------------------------+
+    | "rc"         | :c:enum:`mcumgr_err_t`                        |
+    |              | only appears if non-zero (error condition).   |
+    +--------------+-----------------------------------------------+
+
+Bootloader Information Response
+===============================
+
+Bootloader information response header:
+
+.. table::
+    :align: center
+
+    +--------+--------------+----------------+
+    | ``OP`` | ``Group ID`` | ``Command ID`` |
+    +========+==============+================+
+    | ``2``  | ``0``        |  ``8``         |
```

Comment:
btw. Shouldn't it be `1` - `MGMT_OP_READ_RSP`?

Reply:
Yes, and btw for App Info (already merged) as well:
https://builds.zephyrproject.io/zephyr/pr/59345/docs/services/device_mgmt/smp_groups/smp_group_0.html#os-application-info-response

Reply:
Unaddressed

Reply:
Done